From: Tanu Kaskinen <tanuk@iki.fi>
Date: Sun, 10 Sep 2017 22:32:57 +0300
Subject: sink, source: fix out of date comments

update_shared_real_volume() doesn't exist. I checked that
update_real_volume() does what the comments claim
update_shared_real_volume() to do.

(cherry picked from commit 0062bd860d83d18c34e867594b68de72d94a3259)
---
 src/pulsecore/sink.c   | 2 +-
 src/pulsecore/source.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/pulsecore/sink.c b/src/pulsecore/sink.c
index 2ea874a..2234289 100644
--- a/src/pulsecore/sink.c
+++ b/src/pulsecore/sink.c
@@ -1956,7 +1956,7 @@ static void propagate_reference_volume(pa_sink *s) {
 
             /* Since the origin sink uses volume sharing, this input's volume
              * needs to be updated to match the root sink's real volume, but
-             * that will be done later in update_shared_real_volume(). */
+             * that will be done later in update_real_volume(). */
             continue;
         }
 
diff --git a/src/pulsecore/source.c b/src/pulsecore/source.c
index b9db4c6..e0d32af 100644
--- a/src/pulsecore/source.c
+++ b/src/pulsecore/source.c
@@ -1530,7 +1530,7 @@ static void propagate_reference_volume(pa_source *s) {
 
             /* Since the origin source uses volume sharing, this output's volume
              * needs to be updated to match the root source's real volume, but
-             * that will be done later in update_shared_real_volume(). */
+             * that will be done later in update_real_volume(). */
             continue;
         }
 
